pkg/sac/effectiveaccessscope/effective_access_scope.go

```diff
@@ -62,6 +62,25 @@ func (n *namespacesScopeSubTree) copy() *namespacesScopeSubTree {
 	}
 }
 
+// Cluster is the interface for clusters in the access scope computation
+type Cluster interface {
+	GetId() string
+	GetName() string
+	GetLabels() map[string]string
```

Comment:
That could be modified? But we can handle that in next PR

Reply:
That is correct.
On the other side, the approach should already reduce the risk of involuntarily tampering with the cache content when using data through these interfaces.